### ISSUE TITLE
Add Resourceful Bees once Productive Bees is fully compatible

### ISSUE THREAD

**Issue by NillerMedDild** [NillerMedDild/Enigmatica6#182]
Things needed before we can switch:
- Datapack for bees we want/need
- Check that bees don't breed automatically

**Comment by JaisDK** [comment]
Why though?

**Comment by NillerMedDild** [comment]
Because of the datapack extensibility :)

**Comment by JaisDK** [comment]
I'm adding that in the next release, just got it working last week.

**Comment by NillerMedDild** [comment]
Awesome! In that case I can't think of a reason to switch :D

**Comment by JaisDK** [comment]
The goal is to have a complete "plug and play" mod with support for resources added by other mods and I've come to realize that it's so much easier with datapack support. It will not be as easy to implement as Resourceful Bees because I've split it into several files (one for the bee definition, 1 recipe for breeding, 1 recipe for producing and 1 recipe for comb output), but it will be more customizable that way. I will be using the system myself for most of the bees.

I was thinking to add the following bees. If you have suggestions for more let me know and once the release is out I will gladly help you with the files.

Generel
  - Constantan
  - Obsidian
  
Botania
  - Manasteel
  - Terrasteel
  - Elementium

Mekanism
  - Osmium
  - Refined Obsidian
  - Refined Glowstone

AE2
  - Certus Quartz

Industrial Foregoing
  - Plastic

Quark
  - Biotite

Refined Storage
  - Silicon

**Comment by NillerMedDild** [comment]
Sounds great! I'll be sure to let you know if I come up with other bees.

**Comment by JaisDK** [comment]
Got a version out with the new system and added all but the Plastic, Biotite and Silicon. I'll get them in for the next release.
I've tried balancing the recipes, but if you want to change or add anything and need help doing so, just let me know. I have yet to write a guide for it.

**Comment by NillerMedDild** [comment]
It looks really cool :D
Small note, it seems lang keys/registry names for some of the bees are incorrect:
![billede](https://user-images.githubusercontent.com/21295394/92033278-b3ad8b00-ed6b-11ea-88a7-2bb4073756e9.png)

**Comment by JaisDK** [comment]
Ah shoot, there always have to be something ;)

Well I might as well prepare the next release then and include the last bees.

**Comment by Thiana** [comment]
They also are incorrect in the hive gui. The new bees are all showing as configurable_bee instead of obsidian/etc.

**Comment by NillerMedDild** [comment]
Acacia Nest has a typo in it's display name.

**Comment by NillerMedDild** [comment]
I came up with additional reasons to switch to Resourceful Bees unfortunately, but I'm still team-productive :P 
I'm just gonna list them in case you're interested:
- Tier system for improving beehives, multiblock apiaries
- Many very descriptive JEI pages
- Tooltips with details
- Patchouli book in the works
- Centrifuge texture and GUI

What I think Productive Bees does better:
- Default bees
- Breeding
- Datapack implementation
- Genes

**Comment by JaisDK** [comment]
Thank you for you honesty and feedback. I am not planning on competing fully with Resourceful Bees, in fact I am talking with them about how we can make our mods compatible. I very much like the multiblock apiary and centrifuge from their mod and the plan is to make my bees able to produce in their hives/apiaries and vice versa.

Production wise my bees are certainly lacking behind compare to RB, but where they are basing it on the structure, I am planning on having it based on the bees attributes, a system that's not fully fleshed out yet. I might do upgrades for the advanced hive similar to how Gendustry was doing it, but it's still undecided.

Much more JEI is planned like showing flowering requirements and production chances from combs.

If you're interested I have a mod plan here https://github.com/JDKDigital/productive-bees/projects/1

The vision with compatibility with RB is to be able to pick whichever bees you like from each mod and have them all work the same on each others hives possibly even breed together. It's a few months out though.

**Comment by NillerMedDild** [comment]
Oh that's wonderful, I look forward to using both mods then :D

**Comment by NillerMedDild** [comment]
Sounds great, thanks for popping in :P
